packages/ember-debug/lib/warn.js

```diff
@@ -18,16 +18,6 @@ export let missingOptionsDeprecation = 'When calling `Ember.warn` you ' +
   '`options` should include an `id` property.';
 export let missingOptionsIdDeprecation = 'When calling `Ember.warn` you must provide `id` in options.';
 
-/**
-  Display a warning with the provided message. Ember build tools will
-  remove any calls to `Ember.warn()` when doing a production build.
-
-  @method warn
-  @param {String} message A warning to display.
-  @param {Boolean} test An optional boolean. If falsy, the warning
-    will be displayed.
-  @public
```

Comment:
k, I'll try that and see if it helps.  If it works I'll also do it for depecrecate